src/tests/yamlCompletion.test.ts

```diff
@@ -449,17 +569,6 @@ suite('ASL YAML context-aware completion', () => {
           assert.deepEqual(suggestedSnippets, expectedSnippets)
         })
 
-        test('Does not show state snippets when cursor placed on first line after States prop with same indentation indendation', async () => {
```

Comment:
why do you delete this one? There is some typos in the description that we could fix but we actually should be testing that. If the cursor is at the same level as the `States` prop but on a new line we should verify that we don't get snippets.